cspell.config.yaml

```diff
@@ -1,10 +1,9 @@
 version: "0.2"
 language: en-US
 useGitignore: true
+usePnP: true
 ignorePaths:
   - .vscode
-import:
-  - "@cspell/dict-win32/cspell-ext.json"
```

Comment:
My bad; I failed to consider that this import would still be needed since `@cspell/dict-win32` isn't bundled with CSpell.